### PR TITLE
Initial v2.0.0 upgrade

### PR DESCRIPTION
**Pull request by scottcwilson**
Satisfies db portions of #5601 and #5727

Tested with upgrade and fresh install.

### CODE REVIEW

File: zc_install/sql/install/mysql_zencart.sql [hunk 1]
```diff
@@ -1156,6 +1157,7 @@ CREATE TABLE orders_products (
   orders_id int(11) NOT NULL default '0',
   products_id int(11) NOT NULL default '0',
   products_model varchar(32) default NULL,
+  mpn varchar(32) DEFAULT NULL,
```
Comment (lat9) [inline zc_install/sql/install/mysql_zencart.sql:1160]: Should be `products_mpn`.

File: zc_install/sql/install/mysql_zencart.sql [hunk 2]
```diff
@@ -1620,6 +1622,7 @@ CREATE TABLE products (
   products_type int(11) NOT NULL default '1',
   products_quantity float NOT NULL default '0',
   products_model varchar(32) default NULL,
+  mpn varchar(32) DEFAULT NULL,
```
Comment (lat9) [inline zc_install/sql/install/mysql_zencart.sql:1625]: Should be `products_mpn`.

**Review by lat9 — CHANGES_REQUESTED** [review]
Two name changes

File: includes/version.php [hunk 3]
```diff
@@ -14,8 +14,8 @@
  */
 
 define('PROJECT_VERSION_NAME', 'Zen Cart');
-define('PROJECT_VERSION_MAJOR', '1');
-define('PROJECT_VERSION_MINOR', '5.8a');
+define('PROJECT_VERSION_MAJOR', '2');
+define('PROJECT_VERSION_MINOR', '0.0');
```
Comment (lat9) [inline includes/version.php:18]: Shouldn't this be `0.0-alpha1`?

Comment (scottcwilson) [reply includes/version.php:18]: I'd rather not go down that road unless the group insists.  We're trying to move away from letters, and this is one more thing that has to be remembered on release date that has been messed up in the past.

File: zc_install/sql/install/mysql_zencart.sql (hunk 2 above)
Comment (scottcwilson) [reply zc_install/sql/install/mysql_zencart.sql:1625]: Yep.  TY.

File: zc_install/sql/install/mysql_zencart.sql (hunk 1 above)
Comment (scottcwilson) [reply zc_install/sql/install/mysql_zencart.sql]: Yep.  TY.

File: includes/version.php (hunk 3 above)
Comment (lat9) [reply includes/version.php:18]: That's part of the standardized versioning.  As we move through alpha/beta versions of v2.0.0, I believe that the alpha/beta designation is quite important.

**Comment by drbyte** [comment]
I agree that this needs to be named "-alpha" in the db.

Since it touches several files, I will do it as a separate PR after merging.

**Comment by torvista** [comment]
As of today, version check is incorrect
image

**Pull request merged**